rundeck-scripts/propose

```diff
@@ -4,8 +4,7 @@ source "$(dirname $0)/functions"
 
 echo "Proposing"
 ret=0
-output="$(docker exec rnode \
-	./bin/rnode -c /var/lib/rnode-static/rnode.conf propose 2>&1)" || ret=$?
+output="$(python3.6 $INSTALL_DIR/scripts/propose.py 2>&1)" || ret=$?
```

Comment:
This (and also the previous version) can also fail when there are no deploys, The condition below checks for this failure case (`*NoNewDeploys*`) and resets exit code (status) to 0 (success) in that case. That is because this runs periodically and no new deploys are expected.

You should handle this failure in the script (`propose.py`) so that it finishes successfully when `*NoNewDeploys*` error is returned. ~~From what I gather the `propose.py` doesn't process gRPC reply at all so it will never print something that would match `*NoNewDeploys*`.~~ OK, the `client.propose()` will throw an exception and `NoNewDeploys` will probably appear on stderr. Still it'd be more reliable to catch it in the script.

Reply:
https://github.com/rchain/rchain-testnet-node/pull/17/commits/5f79e0b718850ccb92b9f44e0faabbc85c672777